coderd/wsbuilder/wsbuilder.go

```diff
@@ -427,7 +427,10 @@ func (b *Builder) buildTx(authFunc func(action policy.Action, object rbac.Object
 			},
 			// appease the exhaustruct linter
 			// TODO: set this to whether the build included a `coder_ai_task` tf resource
-			HasAITask: false,
+			HasAITask: sql.NullBool{
+				Bool:  false,
+				Valid: false,
```

Comment:
Does `exhaustruct` still require nillable fields to be set?

```suggestion
			HasAITask: sql.NullBool{
				Bool:  false,
				Valid: true,
```